tests/integration/cmor/_fixes/access/test_access_esm.py

```diff
@@ -0,0 +1,260 @@
+                            {'latitude': coord_info})
+
+        cubes = cubes_2d
+        fixed_cubes = fix.fix_metadata(cubes)
+
+        # Check cube metadata
+        if var == 'tas':
+            cube = check_tas_metadata(fixed_cubes)
+        elif var == 'pr':
+            cube = check_pr_metadata(fixed_cubes)
+
+        # Check cube data
+        assert cube.shape == (1, 145, 192)
+
+        # Check latitude metadata
+        assert cube.coords('latitude', dim_coords=True)
+        new_lat_coord = cube.coord('latitude')
+        assert new_lat_coord.var_name == 'lat'
+        assert new_lat_coord.standard_name == 'latitude'
+        assert new_lat_coord.units == 'degrees_north'
+
+
+def test_only_longitude(monkeypatch, cubes_2d):
+    """Test fix."""
+    var_list = ['tas', 'pr']
+
+    for var in var_list:
+        fix = get_fix('Amon', 'mon', var)
+
+        coord_info = CoordinateInfo('longitude')
+        coord_info.standard_name = 'longitude'
+        monkeypatch.setattr(fix.vardef, 'coordinates',
+                            {'longitude': coord_info})
+
+        cubes = cubes_2d
+        fixed_cubes = fix.fix_metadata(cubes)
+
+        # Check cube metadata
+        if var == 'tas':
+            cube = check_tas_metadata(fixed_cubes)
+        elif var == 'pr':
+            cube = check_pr_metadata(fixed_cubes)
+
+        # Check cube data
+        assert cube.shape == (1, 145, 192)
+
+        # Check longitude metadata
+        assert cube.coords('longitude', dim_coords=True)
+        new_lon_coord = cube.coord('longitude')
+        assert new_lon_coord.var_name == 'lon'
+        assert new_lon_coord.standard_name == 'longitude'
+        assert new_lon_coord.units == 'degrees_east'
+
+
+def test_get_tas_fix():
+    """Test getting of fix 'tas'."""
+    fix = Fix.get_fixes('ACCESS', 'ACCESS_ESM', 'Amon', 'tas')
+    assert fix == [
+        esmvalcore.cmor._fixes.access.access_esm.Tas(vardef={},
+                                                     extra_facets={},
+                                                     session={},
+                                                     frequency=''),
+        GenericFix(None),
+    ]
+
+
+def test_get_pr_fix():
+    """Test getting of fix 'pr'."""
+    fix = Fix.get_fixes('ACCESS', 'ACCESS_ESM', 'Amon', 'pr')
+    assert fix == [
+        esmvalcore.cmor._fixes.access.access_esm.Pr(vardef={},
+                                                    extra_facets={},
+                                                    session={},
+                                                    frequency=''),
+        GenericFix(None),
+    ]
+
+
+def test_tas_fix(cubes_2d):
+    """Test fix 'tas'."""
+    fix = get_fix('Amon', 'mon', 'tas')
+    fixed_cubes = fix.fix_metadata(cubes_2d)
+    fixed_cube = check_tas_metadata(fixed_cubes)
+
+    check_time(fixed_cube)
+    check_lat(fixed_cube)
+    check_lon(fixed_cube)
+    check_heightxm(fixed_cube, 1.5)
```

Comment:
[`tas`](https://github.com/ESMValGroup/ESMValCore/blob/ee16a958d916f1620a2aa25191e4cc34b3886006/esmvalcore/cmor/tables/cmip6/Tables/CMIP6_Amon.json#L1151-L1168) is usually the 2m temperature. Is there a more suitable variable available in the raw model output? If not, and you don't care about the discrepancy, I would set the value to 2 in the fix (otherwise comparing this with output from other models might be problematic). If you care, you might have to interpolate.